winter/pagination/page_position_argument_inspector.py

```diff
@@ -27,11 +29,31 @@ def __init__(self, page_position_argument_resolver: PagePositionArgumentResolver
             in_=openapi.IN_QUERY,
             type=openapi.TYPE_INTEGER,
         )
+        self.default_order_by_parameter = openapi.Parameter(
+            name=page_position_argument_resolver.order_by_name,
+            description='Comma separated order by fields',
+            required=False,
+            in_=openapi.IN_QUERY,
+            type=openapi.TYPE_STRING,
+        )
 
     def inspect_parameters(self, route: 'Route') -> List[openapi.Parameter]:
         parameters = []
+        allowed_order_by_fields = get_allowed_order_by_fields(route.method)
+        if not allowed_order_by_fields:
+            order_by_parameter = self.default_order_by_parameter
+        else:
+            order_by_parameter = openapi.Parameter(
+                name=self._page_position_argument_resolver.order_by_name,
+                description='Comma separated order by fields',
+                required=False,
+                in_=openapi.IN_QUERY,
+                type=openapi.TYPE_STRING,
```

Comment:
here is string not list of string, because now drf yasg do not support enums for array, object or None. And I think it is better to use string with enum than array without enum

Reply:
Ok, let's enumerate fields in description.